utils/rpms/packaging/get_release_branch

```diff
@@ -3,10 +3,9 @@
 # find the base branch of the current branch
 # base branches can be master, release/2.4+, release/3+
 # or optionally branches passed into $1
-
 set -eu -o pipefail
 IFS=' ' read -r -a add_bases <<< "${1:-}"
-origin=origin
+origin="${ORIGIN:-origin}"
```

Comment:
So yeah.  This change really needs to be made in https://github.com/daos-stack/packaging/ but indeed, I am seeing that `get_release_branch` doesn't even exist there yet.  Something for me to keep in mind for a future update of that repo I suppose.